bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/dialogfields/Multifield.java

```diff
@@ -65,6 +66,19 @@ public void setValue(Object value) {
   }
 
   /**
+   * Returns MultifieldItem at declared index position
+   *
+   * @param index integer representing required position
+   * @return MultifieldItem
+   */
+  public MultifieldItem getItemAtIndex(int index) {
+    if (items.size() > index) {
+      return items.get(index);
+    }
+    throw new BobcatRuntimeException("Trying to reach nonexisting item.");
```

Comment:
Maybe smth like: "Tried to get item at index `n` but there are only `k` elements" Where n=index and k=items.size()